plugin/src/com/microsoft/alm/plugin/external/ToolRunner.java

```diff
@@ -385,7 +386,7 @@ public void run() {
             // Don't let exceptions escape from this top level method
             try {
                 // Create a buffered reader so that we can process the output one line at a time
-                bufferedReader = new BufferedReader(new InputStreamReader(stream));
+                bufferedReader = new BufferedReader(new InputStreamReader(stream,StandardCharsets.UTF_8));
```

Comment:
Space is missing after the comma

```suggestion
                bufferedReader = new BufferedReader(new InputStreamReader(stream, StandardCharsets.UTF_8));
```